README.md

```diff
@@ -16,69 +16,60 @@ Right now its responsibilities cover:
 
 Example: (Updated)
 
-# Quick start
+# Quick start with Buddy cli
 
-# To run your code locally
-
-Start by forking the example repo https://github.com/ministry-of-silly-code/examples
+## Requirements
 
-```shell
-# 1 - Create a new virtual env
-python3 -m venv venv
-source ./venv/bin/activate
+1. You now need to setup the cluster SSH key. how create and add your SSH keys can be found [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account) 
+1. Add cluster public-key to: `https://github.com/settings/keys` if you don't know this yet, check this out [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
+1. `pip install wandb && wandb init` there are two ways to set up wandb: 
+    - set up wandb init from the cmd
+    - or set export  WANDB_API_KEY = your wandb key which can be found here: https://wandb.ai/settings
 
-# 2 - Clone [or create] your project, in this case I'm using a basic mnist_classifier 
-git clone https://github.com/[your-username]/examples.git
-cd examples
 
-# 3 - Install Buddy
-# (and add it to you your requirements.txt if you are not using the given examples)
-pip install -e git+https://github.com/ministry-of-silly-code/experiment_buddy.git#egg=experiment_buddy
+## Installation
+### From repo
+```shell
+git clone git@github.com:ministry-of-silly-code/buddy_cli.git
+pip install -e ./buddy-cli
```

Comment:
this will install buddy in edit mode and  global, i'm not sure is something we want to do. Either the user should have a venv prior to buddy, or `pip install --user ./buddy-cli` which at least is in `~/.local` instead of global path.

It is our responsibility then to keep the dependencies at the bare minimum.

Reply:
This piece is for the development, when we get out from the beta testing, this will disappear and PyPi will be used instead, but it's useful for me to have it around for now